basecoin/modules/src/ibc/impls.rs

```diff
@@ -75,6 +75,12 @@ pub enum AnyConsensusState {
     Tendermint(TmConsensusState),
 }
 
+impl From<ConsensusStateType> for AnyConsensusState {
```

Comment:
Just curious: how come we don't require an `impl From<ClientStateType> for AnyClientState`?

Reply:
we don't have that `AnyClientState` enum in basecoin-rs at the moment. Instead, it's rolling with `TmClientState` as the associated type.